docs/policy/rdt.md

```diff
@@ -104,124 +104,52 @@ Besteffort, Besteffort getting only 50% of this. Guaranteed class gets full
+      l3Allocation: "40%"
+      mbAllocation: ["50%"]
+      classes:
+        Burstable:
+          # Allow "Burstable" to use all cache lines of the "shared" partition
+          l3Allocation: "100%"
+        BestEffort:
+          # Allow "Besteffort" to use only half of the L3 cache # lines of the "shared" partition.
+          # These will overlap with those used by "Burstable"
+          l3Allocation: "50%"
 ```
 
 The configuration also supports far more fine-grained control, e.g. per
 cache-ID configuration (i.e. different sockets having different allocation) and
 Code and Data Prioritization (CDP) allowing different cache allocation for code
-and data paths.
-
-```yaml
-...
-    partitions:
-      exclusive:
-        l3Allocation: "60%"
-        mbAllocation: ["100%"]
-        classes:
-          # Automatically gets 100% of what was allocated for the partition
-          Guaranteed:
-      shared:
-        l3Allocation:
-          # 'all' denotes the default and must be specified
-          all: "40%"
-          # Specific cache allocation for cache-ids 2 and 3
-          2-3: "20%"
-        mbAllocation: ["100%"]
-        classes:
-          Burstable:
-            l3Schema:
-              all:
-                unified: "100%"
-                code: "100%"
-                data: "80%"
-            mbSchema:
-              all: ["80%"]
-              2-3: ["50%"]
-...
-...
-```
-
-In addition, if the hardware details are known, raw bitmasks or bit numbers
+and data paths. If the hardware details are known, raw bitmasks or bit numbers
 ("0x1f" or 0-4) can be used instead of percentages in order to be able to
-configure cache allocations exactly as required. The bits in this case are
-corresponding to those in `/sys/fs/resctrl/` bitmasks. You can also mix relative
-(percentage) and absolute (bitmask) allocations. For cases where the resctrl
-filesystem is mounted with `-o mba_MBps` Memory bandwidth must be specifed in
-MegaBytes per second (MBps).
-
-```yaml
-...
-    partitions:
-      exclusive:
-        # Specify bitmask in bit numbers
-        l3Allocation: "8-19"
-        # MBps value takes effect when resctrl mount option mba_MBps is used
-        mbAllocation: ["100%", "100000MBps"]
-        classes:
-          # Automatically gets 100% of what was allocated for the partition
-          Guaranteed:
-      shared:
-        # Explicit bitmask
-        l3Allocation: "0xff"
-        mbAllocation: ["50%", "2000MBps"]
-        classes:
-          # Burstable gets 100% of what was allocated for the partition
-          Burstable:
-          BestEffort:
-            l3Schema: "50%"
-            # Besteffort gets 50% of the 50% (i.e. 25% of total) or 1000MBps
-            mbSchema: ["50%", "1000MBps"]
-```
+configure cache allocations exactly as required. For detailed description of the RDT configuration format with examples see the
+[goresctrl library documentation](https://github.com/intel/goresctrl/blob/v0.2.0/doc/rdt.md)
```

Comment:
Why not point the link to latest version `https://github.com/intel/goresctrl/blob/main/doc/rdt.md`, that way this documentation is always "correct" one and we do not need to adjust it when new goresctrl version are created.

Reply:
Good question. I was thinking about this but decided that we better refer to the version that we actually depend on. Future versions of goresctrl may have new features that we don't support (or even have incompatible changes, who knows) which would confuse users. I now added a patch that specifies the goresctrl version in sphinx config which hopefully makes the maintenance pain little smaller.